packages/@react-aria/interactions/src/utils.ts

```diff
@@ -30,6 +30,18 @@ export function isVirtualClick(event: MouseEvent | PointerEvent): boolean {
     return true;
   }
 
+  // Android TalkBack's detail value varies depending on the event listener providing the event so we have specific logic here instead
+  // If pointerType is defined, event is from a pointer or click listener.
+  // For events from mousedown listener, detail === 0 is a sufficient check to detect TalkBack virtual clicks
+  if (isAndroid() && (event as PointerEvent).pointerType != null) {
+    if (event.type === 'pointerdown') {
+      let {width, height, pressure} = event as PointerEvent;
+      return width === 1 && height === 1 && pressure === 0;
+    }
+
+    return event.type === 'click' && event.buttons === 1;
+  }
+
```

Comment:
Updated isVirtualClick's detection of Android Talkback clicks, I can provide diffs comparing the events of tap vs Talkback virtual click if people are curious

Reply:
At the moment `isVirtualClick` isn't called with a PointerEvent, the above change was more of a defensive change than anything in case we used `isVirtualClick` in a pointer listener. `isVirtualPointerEvent` is indeed the exact same check, but this one was pared down since the `isAndroid()` check means we don't have to do the extra checks to distinguish it from FF touch/iOS. With #3524 making `isVirtualPointerEvent` reusable else where, I'm happy to remove the `pointerdown` portion of the logic here